### PR TITLE
Simplify beta hats

### PR DESCRIPTION
**Pull request by davidbenjamin**
@kareembroad @vruano @achevali @LeeTL1220 This is what I was talking about yesterday.  It turns out that the entire beta hats code, aside from filtering out low-coverage targets, is a one-line matrix multiplication.

### CODE REVIEW

**Comment by davidbenjamin** [comment]
@LeeTL1220 Please disregard the error on Travis.  It's dealt with in PR #241.

**Comment by LeeTL1220** [comment]
@davidbenjamin This change could be merged into my branch... I misunderstood from previous conversation.

**Comment by LeeTL1220** [comment]
@davidbenjamin A simple unit test to show that having irrelevant targets does not break this would be useful (as per our conversation).

**Comment by davidbenjamin** [comment]
@LeeTL1220 test added.  I put it in `NormalizeSomaticReadCountsIntegrtionTest` even though that's not totally logical because that's where all the other PoN methods are tested.  Let me know if I should move it.

**Comment by LeeTL1220** [comment]
That is fine.

File: src/test/java/org/broadinstitute/hellbender/tools/exome/NormalizeSomaticReadCountsIntegrationTest.java [hunk 1]
```diff
@@ -508,6 +512,41 @@ private void assertPreTangentNormalizedValues(final ReadCountCollection factorNo
     }
 
     /**
+     * Asserts that the calculation of beta hats is not significantly affected by zero-coverage outlier counts
+     * We perform this check by randomly setting
```
Comment (LeeTL1220) [inline src/test/java/org/broadinstitute/hellbender/tools/exome/NormalizeSomaticReadCountsIntegrationTest.java]: Can you finish this sentence?

**Comment by LeeTL1220** [comment]
@davidbenjamin Trivial change....

**Comment by LeeTL1220** [comment]
@davidbenjamin After correcting, feel free to merge...

**Pull request merged**